config-model-api/src/main/java/com/yahoo/config/model/api/ModelContext.java

```diff
@@ -37,7 +37,7 @@
     default Optional<File> appDir() { return Optional.empty();}
 
     /** The Docker image repo we want to use for images for this deployment (optional, will use default if empty) */
-    default Optional<DockerImage> wantedDockerImageRepo() { return Optional.empty(); }
```

Comment:
This breaks the API between the config server and config model. You need to keep the old method with a TODO specifying when it can be removed . Unfortunately, since I think we have pinned applications now that should be by version and not just something like "Remove in October 2020".